solc/CommandLineInterface.cpp

```diff
@@ -716,6 +763,12 @@ void CommandLineInterface::processInput()
 		compile();
 		outputCompilationResults();
 		break;
+	case InputMode::EVMAssemblerJSON:
+		assembleFromEVMAssemblyJSON();
+		handleCombinedJSON();
+		handleBytecode(m_assemblyStack->contractNames().front());
+		handleEVMAssembly(m_assemblyStack->contractNames().front());
+		break;
```

Comment:
In the compiler mode the compiler will print `"Compiler run successful. No output generated."` if there is no output. This does not happen for assembly import. Looks like that's because the code to handle it sits in `outputCompilationResults()` and here we're bypassing it.